app/src/main/java/com/trianglz/weatherapp/data/remotesource/IRemoteDataSource.kt

```diff
@@ -0,0 +1,27 @@
+package com.trianglz.weatherapp.data.remotesource
+
+import com.trianglz.weatherapp.data.models.city.CityDto
+import com.trianglz.weatherapp.data.models.country.CountryDto
+import com.trianglz.weatherapp.data.models.weather.WeatherDto
+
+interface IRemoteDataSource {
```

Comment:
Our convention also, is to have the interface named like CitiesRemoteSource. and it's implementer will be CitiesRemoteSourceImpl